e2e/pages/createClaim/paymentReference.page.js

```diff
@@ -0,0 +1,16 @@
+const {I} = inject();
+
+module.exports = {
+
+  fields: {
+    paymentReference: '#paymentReference',
+  },
+
+  async updatePaymentReference() {
+    I.waitForElement(this.fields.paymentReference);
+    I.clearField(this.fields.paymentReference);
```

Comment:
why is this required?

Reply:
To be honest, it isn't. The field is prepopulated from solicitor references, the clear was so that we type in new text